shared/js/background/events.es6.js

```diff
@@ -358,7 +358,7 @@ browser.runtime.onMessage.addListener((req, sender) => {
     if (req.messageType && req.messageType in messageHandlers) {
         return Promise.resolve(messageHandlers[req.messageType](req.options, sender))
     }
-    if (req.messageType === 'registeredContentScript' || req.registeredTempAutofillContentScript) {
+    if (req.messageType === 'registeredContentScript' || req.messageType === 'registeredAutofillContentScript') {
         const argumentsObject = getArgumentsObject(sender.tab.id, sender, req.options?.documentUrl || req.documentUrl, sessionKey)
```

Comment:
@alistairjcbrown are we able to nest inside the options key, we do that for all other messages hence the other weird wart here too.
Sorry should have specified!